pkg/controllers/ippool/ippool_controller.go

```diff
@@ -114,6 +116,8 @@ func (r *IPPoolReconciler) setReadyStatusTrue(ctx *context.Context, ippool *v1al
 }
 
 func (r *IPPoolReconciler) Reconcile(ctx context.Context, req ctrl.Request) (ctrl.Result, error) {
```

Comment:
So you mean that in the active pod, the NSX operator runs the Reconcile; in the standby pod, the NSX operator doesn't run the Reconcile, right? Could you please point out the logic to control that? I want to check it also.

Reply:
Yes, from the log
```
2024-06-14 05:24:29.170	INFO	starting server	{"kind": "health probe", "addr": "[::]:8385"}
I0614 05:24:29.171301  340277 leaderelection.go:250] attempting to acquire leader lease default/nsx-operator...
I0614 05:24:29.201179  340277 leaderelection.go:260] successfully acquired lease default/nsx-operator
2024-06-14 05:24:29.201	DEBUG	sc2-hs2-nsbu-ncp-dhcp-83-146_31a75034-77ea-4c8c-a087-a868ea856207 became leader	{"type": "Normal", "object": {"kind":"Lease","namespace":"default","name":"nsx-operator","uid":"d571880d-fcfc-4aca-b832-81e0a21199cc","apiVersion":"coordination.k8s.io/v1","resourceVersion":"10266376"}, "reason": "LeaderElection"}
2024-06-14 05:24:29.201	INFO	Starting EventSource	{"controller": "networkinfo", "controllerGroup": "nsx.vmware.com", "controllerKind": "NetworkInfo", "source": "kind source: *v1alpha1.NetworkInfo"}
2024-06-14 05:24:29.201	INFO	Starting EventSource	{"controller": "networkinfo", "controllerGroup": "nsx.vmware.com", "controllerKind": "NetworkInfo", "source": "kind source: *v1alpha1.VPCNetworkConfiguration"}
2024-06-14 05:24:29.201	INFO	Starting Controller	{"controller": "networkinfo", "controllerGroup": "nsx.vmware.com", "controllerKind": "NetworkInfo"}
```
leaderelection.go controllers it.

Reply:
So, if the pod is running from active to standby, do we see logs to show Reconcile is stopped? 
From code logic, we see GC is started in start manger function as a goroutine. So, gorouting is still running even the pod in standby mod? Do we have log to show how the pod behaves in standby mode?

Reply:
As we discussed, the process would crash if it transitions to standby mode.